game-service/src/main/java/org/libertybikes/game/core/GameBoard.java

```diff
@@ -139,4 +142,58 @@ public boolean moveObjects() {
         return true;
     }
 
+    public boolean broadcastToAI() {
+        if (ai.isEmpty()) {
+            return false;
+        }
+
+        for (AIPlayer p : ai) {
+            try {
+                p.broadCastBoard(board);
+            } catch (Exception e) {
+                // bad ai
```

Comment:
we should at least log a message or stack trace here instead of swallowing the exception completely